### DISCUSSION TITLE
post/a55b5348

### DISCUSSION THREAD

**Discussion by giscus** [ZhangStudyLife/ZhangStudyLife.github.io#2] [Show and tell]
# post/a55b5348

我的第一篇博客这篇博文仅供我初学部署网页以及md语法使用望众大佬嘴下留情 2024-8-4 00:21：一级标题二级标题三级标题空格第一排句子后面没有空格第二排换行了我，而且这行有空格第三排我换行了告诉我要有按空格的好习惯对吧   强调接下来是强调加上两个星号我要开始强调了：我还没有开始强调呢我强调了这句话 斜体一个星号包裹起来是斜体：这里应该是斜的吧 大成之作听说三个星号是集大成之作：强调加上斜

http://localhost:4000/post/a55b5348.html

**Comment by ZhangStudyLife** [comment]
我像个 小丑  
改了半天配置  
结果搞不好  
还求助Ruakker  
***然后发现改的是我保存在桌面的备份的配置文件。。。***

**Comment by Ruakker** [comment]
🤡

**Comment by ZhangStudyLife** [comment]
🤡
